### PR TITLE
Leaving game - unmute players

### PR DESCRIPTION
**Pull request by dpieve**
I noticed that when we use the command 'endgame' the players kept muted. Then I'm adding this code to fix that. I didn't test.

### CODE REVIEW

File: src/GameManager.js [hunk 1]
```diff
@@ -15,6 +15,10 @@ class GameManager {
   }
 
   endGame (guildId) {
+    Game.players.forEach(player => {
```
Comment (pedrofracassi) [inline src/GameManager.js]: That wouldn't work. You'd have to `this.getGame(guildId)` instead of using plain `Game`

**Review by pedrofracassi — CHANGES_REQUESTED** [review]

**Comment by pedrofracassi** [comment]
Nice catch! I've requested some changes, if you could please take a look :)

**Comment by dpieve** [comment]
Changes done, thanks

**Comment by pedrofracassi** [comment]
Thank you so much for the PR. I'm going to close this as I ended up implementing it myself after some major code changes.

**Pull request closed without merging**